tests/src/Interop/ReversePInvoke/Marshalling/CMakeLists.txt

```diff
@@ -7,6 +7,4 @@ set(SOURCES MarshalBoolArrayDLL.cpp MarshalBoolArrayDLL.def)
 add_library (MarshalBoolArrayDLL SHARED ${SOURCES})
 
 # add the install targets
-install (TARGETS MarshalBoolArrayDLL DESTINATION bin)
-
-
+install (TARGETS MarshalBoolArrayDLL DESTINATION bin)
```

Comment:
It is done

From: Aditya Mandaleeka [mailto:notifications@github.com]
Sent: Friday, November 13, 2015 2:57 PM
To: dotnet/coreclr
Cc: Will Godbe
Subject: Re: [coreclr] Updated lists of failing/unsupported tests outside of Windows (#2048)


In tests/src/Interop/ReversePInvoke/Marshalling/CMakeLists.txt<https://github.com/dotnet/coreclr/pull/2048#discussion_r44844105>:

> @@ -7,6 +7,4 @@ set(SOURCES MarshalBoolArrayDLL.cpp MarshalBoolArrayDLL.def)

>  add_library (MarshalBoolArrayDLL SHARED ${SOURCES})

>

>  # add the install targets

> -install (TARGETS MarshalBoolArrayDLL DESTINATION bin)

> -

> -

> +install (TARGETS MarshalBoolArrayDLL DESTINATION bin)

Looks like nothing really changed in this file. Can you just undo this change?

—
Reply to this email directly or view it on GitHub<https://github.com/dotnet/coreclr/pull/2048/files#r44844105>.